internal/cluster/cmd/clusterd/main.go

```diff
@@ -127,6 +127,9 @@ func main() {
 	case "v2":
 		n.HardforkV2.AllowHeight = 2
 		n.HardforkV2.RequireHeight = 3
+	case "transition":
```

Comment:
I noticed that the cluster mines until block 800 almost immediately after startup and waiting for contracts. Maybe we should put the hardfork height after that? Or check why it mines 800 blocks to begin with. I know we mine until maturity height but that doesn't account for all blocks

Reply:
Oh really, I settled on these heights because it seems to stop mining around low 300s for me - this has been working. Not sure why it would be different for you, does cmd/clusterd do something different?